venvs/common.py

```diff
@@ -56,8 +71,20 @@ def exists_on(self, filesystem):
     def binary(self, name):
         return self.path.descendant("bin", name)
 
-    def create(self, arguments=(), stdout=sys.stdout, stderr=sys.stderr):
-        self._create(self, arguments=arguments, stdout=stdout, stderr=stderr)
+    def create(
+            self,
+            arguments=(),
+            python=None,
```

Comment:
Think the default to `sys.executable` seems like it might belong here rather than in the lower level implementation.

Reply:
Then any substitute creator callable can't set its own default (just thought of this now).  But sure, changing now.